### PR TITLE
params: implement ECIP-1078/1079 feature activation

### PR DESCRIPTION
**Pull request by meowsbits**
> - https://github.com/ethereumclassic/ECIPs/pull/263/files?short_path=d39459a#diff-d39459a451d1f1cf1bb1d6b1fddcb4f0
> - https://github.com/ethereumclassic/ECIPs/pull/261/files#diff-f331ca50d292642838a6be87214ac1e7

Implements 'Phoenix' aka 'Aztlan Fix' fork specification
which is described as a 'simultaneous' fork adjacent
to the fork specified in ECIP-1061 and ECIP-1072.

The intention is to sidestep potential vulnerabilities
related to gas costs.

Signed-off-by: meows <b5c6@protonmail.com>

### CODE REVIEW

**Comment by soc1c** [comment]
FYI https://github.com/ethereumclassic/ECIPs/issues/284

**Review by soc1c — COMMENTED** [review]
We need to explicitly disable EIP 2200. On Mordor it already triggered and on Kotti it will soon. For mainnet, we will decide next week.

File: params/config_mordor.go [hunk 1]
```diff
@@ -57,12 +57,16 @@ var (
...
 		// Istanbul eq
+		// ECIP-1061, ECIP-1072
 		EIP152FBlock:  big.NewInt(778507),
 		EIP1108FBlock: big.NewInt(778507),
 		EIP1344FBlock: big.NewInt(778507),
-		EIP1884FBlock: nil,
 		EIP2028FBlock: big.NewInt(778507),
-		EIP2200FBlock: big.NewInt(778507), // RePetersburg (== re-1283)
+		// ECIP-1078, ECIP-1079
+		//EIP2200FBlock: big.NewInt(778507), // RePetersburg (== re-1283)
```
Comment (soc1c) [inline params/config_mordor.go]: this won't work for mordor anymore.

**Comment by soc1c** [comment]
proposed block numbers for phoenix/ecip-1078
- `976_231` on Mordor Classic PoW-testnet (March 4th, 2020)
- `2_208_203` on Kotti Classic PoA-testnet (March 11th, 2020)
- `10_500_839` on Ethereum Classic PoW-mainnet (March 25th, 2020)

ref https://github.com/ethereumclassic/ECIPs/pull/285

**Review by soc1c — CHANGES_REQUESTED** [review]
- needs to disable eip-2200 on mordor and kotti
- needs to update block numbers for phoenix/ecip-1078

**Comment by meowsbits** [comment]
@soc1c Thanks for review. I need to merge some stuff first, but then will address these things soon!

File: params/config_kotti.go [hunk 2]
```diff
@@ -63,13 +63,19 @@ var (
 		EIP1052FBlock: big.NewInt(1705549),
 
 		// Istanbul eq
+		// ECIP-1061, ECIP-1072
 		EIP152FBlock:  big.NewInt(2058191),
 		EIP1108FBlock: big.NewInt(2058191),
 		EIP1344FBlock: big.NewInt(2058191),
-		EIP1884FBlock: nil,
```
Comment (soc1c) [inline params/config_kotti.go:69]: can we keep this line just for the sake of readability? I know it does not solve anything.kk

File: params/config_classic.go [hunk 3]
```diff
@@ -60,14 +60,18 @@ var (
...
-		EIP1344FBlock: big.NewInt(10500839),
-		EIP1884FBlock: nil,
-		EIP2028FBlock: big.NewInt(10500839),
-		EIP2200FBlock: big.NewInt(10500839), // RePetersburg (=~ re-1283)
+		// ECIP-1061, ECIP-1072
+		EIP152FBlock:  big.NewInt(10_500_839),
+		EIP1108FBlock: big.NewInt(10_500_839),
+		EIP1344FBlock: big.NewInt(10_500_839),
+		EIP2028FBlock: big.NewInt(10_500_839),
+		//EIP2200FBlock: big.NewInt(10500839), // RePetersburg (=~ re-1283)
```
Comment (soc1c) [inline params/config_classic.go]: can we have enable and disable on the same block number to be super correct and improve readability? I know it wouldn't change anything.

**Review by soc1c — APPROVED** [review]
logic looks good to me. left two comments for cosmetic improvements.

File: params/config_kotti.go (hunk 2 above)
Comment (meowsbits) [reply params/config_kotti.go:69]: https://github.com/etclabscore/multi-geth/pull/128/commits/26b180edc4edb435ee8f68c08b118329dcdf87e9

File: params/config_classic.go (hunk 3 above)
Comment (meowsbits) [reply params/config_classic.go]: https://github.com/etclabscore/multi-geth/pull/128/commits/26b180edc4edb435ee8f68c08b118329dcdf87e9

**Review by tzdybal — COMMENTED** [review]
Better late than never - LGTM

**Pull request merged**